docs/platforms/react-native/sourcemaps/uploading/codepush.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: CodePush
+sidebar_order: 100
+description: "Upload source maps for CodePush releases."
+---
+
+Sentry's React Native SDK works out of the box with CodePush. To see readable stack traces in the product, you must upload source maps to Sentry. This guide explains how to upload source maps for CodePush releases, that are created with the AppCenter CLI.
+
+## Prerequisities
```

Comment:
```suggestion
## Prerequisites
```

Reply:
@lizokm I saw many pages on the docs using `Prerequisities`, should we keep it as is?

Reply:
Thank you @lucas-zimerman, that's a typo I introduced. 😅

Reply:
No worries, there were so many pages on docs using the same typo that I was wondering if that was actually correct